### DISCUSSION TITLE
Multiple sticks for single z2m instance - zigbee2mqtt in cluster

### DISCUSSION THREAD

**Discussion by bkupidura** [Koenkk/zigbee2mqtt#8098] [General]
Im running z2m as K8s POD. Currently my container is running always on same node (to which ConbeeII is connected).

Anybody tried to connect multiple sticks to different nodes and spawn z2m on one of them?

So there will be always *1* instance of z2m running, but this instance can float between nodes. 
Z2M data files (configuration, state.json, anything else) can be served from shared storage.

Anybody knows if we are storying anything crucial on stick itself? 
What will happend if i replace stickA with stickB (same stick hardware and firmware) and restart z2m?

**Comment by reski-rukmantiyo** [comment]
hi @bkupidura , i'm trying to create the same use case. currently use 2 z2m but connect to 1 mqtt.
the problem is that home assistant only take 1 active z2m. dunnow why.
any experiences of these?

**Reply by bkupidura** [reply]
If you run 2 copies of z2m, you would endup with 2 different zigbee networks... End-device is capable to connect to only single zigbee network.
